src/nls/it/strings.js

```diff
@@ -648,7 +648,104 @@ define({
+    "DESCRIPTION_JSLINT_OPTIONS_COUCH"               : "Vero se CouchDB globali devono essere predefiniti",
+    "DESCRIPTION_JSLINT_OPTIONS_DEBUG"               : "Vero per consentire dichiarazioni debugger",
+    "DESCRIPTION_JSLINT_OPTIONS_DEVEL"               : "Vero se globali del browser utili in fase di sviluppo devono essere predefiniti",
+    "DESCRIPTION_JSLINT_OPTIONS_EQEQ"                : "Vero per consentire == e !=",
+    "DESCRIPTION_JSLINT_OPTIONS_ES6"                 : "Vero se ES6 globali dovrebbero essere predefinite",
+    "DESCRIPTION_JSLINT_OPTIONS_EVIL"                : "Vero per consentire eval",
+    "DESCRIPTION_JSLINT_OPTIONS_FORIN"               : "Vero per consentire non filtrato per in",
+    "DESCRIPTION_JSLINT_OPTIONS_INDENT"              : "Imposta una specifica larghezza del tab",
+    "DESCRIPTION_JSLINT_OPTIONS_MAXERR"              : "Il massimo numero di avvertimenti",
+    "DESCRIPTION_JSLINT_OPTIONS_MAXLEN"              : "Il massimo numero di caratteri in una linea",
+    "DESCRIPTION_JSLINT_OPTIONS_NEWCAP"              : "True per consentire costruttori in minuscolo",
+    "DESCRIPTION_JSLINT_OPTIONS_NODE"                : "Vero se le globali Node.js dovrebbero essere predefinite",
+    "DESCRIPTION_JSLINT_OPTIONS_NOMEN"               : "Vero per consentire penzoloni di sottolineatura in identificatori",
+    "DESCRIPTION_JSLINT_OPTIONS_PASSFAIL"            : "Vero per fermarsi al primo errore",
+    "DESCRIPTION_JSLINT_OPTIONS_PLUSPLUS"            : "Vero per consentire ++ e --",
+    "DESCRIPTION_JSLINT_OPTIONS_REGEXP"              : "Vero per consentire . e [^...]. in RegExp",
+    "DESCRIPTION_JSLINT_OPTIONS_RHINO"               : "Vero se globali Rhino devono essere predefiniti",
+    "DESCRIPTION_JSLINT_OPTIONS_SLOPPY"              : "Vero per consentire di omettere `use strict`",
+    "DESCRIPTION_JSLINT_OPTIONS_STUPID"              : "Vero per consentire la stupidità",
+    "DESCRIPTION_JSLINT_OPTIONS_SUB"                 : "Vero per consentire indicizzazione inefficiente",
+    "DESCRIPTION_JSLINT_OPTIONS_TODO"                : "Vero per consentire commenti Cose da fare",
+    "DESCRIPTION_JSLINT_OPTIONS_UNPARAM"             : "Vero per consentire parametri non utilizzati",
+    "DESCRIPTION_JSLINT_OPTIONS_VARS"                : "Vero per consentire a più di 1 var in una funzione",
+    "DESCRIPTION_JSLINT_OPTIONS_WHITE"               : "Vero per ignorare le regole spazio-bianco",
+    "DESCRIPTION_LANGUAGE"                           : "Impostazioni specifiche della lingua",
+    "DESCRIPTION_LANGUAGE_FILE_EXTENSIONS"           : "Ulteriori mappature di estensione a nome di lingua",
+    "DESCRIPTION_LANGUAGE_FILE_NAMES"                : "Ulteriori mappature da nome di file a nome di lingua",
+    "DESCRIPTION_LINTING_ENABLED"                    : "Vero abilità Code Inspection",
+    "DESCRIPTION_ASYNC_TIMEOUT"                      : "Il tempo in millisecondi dopo la pausa del linter asincrono",
```

Comment:
`Il tempo in millisecondi dopo `**`il quale il controllo del codice viene interrotto`**